backends/cuda/ceed-cuda-compile.cpp

```diff
@@ -97,11 +95,19 @@ int CeedCompile_Cuda(Ceed ceed, const char *source, CUmodule *module, const Ceed
     char  *log;
     size_t log_size;
 
+    CeedDebug256(ceed, CEED_DEBUG_COLOR_ERROR, "---------- CEED JIT SOURCE FAILED TO COMPILE ----------\n");
+    CeedDebug(ceed, "File: %s\n", jit_defs_path);
+    CeedDebug(ceed, "Source:\n%s\n", jit_defs_source);
```

Comment:
```suggestion
    CeedDebug(ceed, "Source:\n%s\n", code);
```

I actually printed out `code`, not `jit_defs_source`. Not sure how much of a difference that makes, but might as well have the full buffer just in case?

Reply:
Ah, duh. Silly mistake on my part